apps/web/src/server/views/inspector/book-site-visit-success.njk

```diff
@@ -0,0 +1,30 @@
+{% extends "app/layouts/app-two-column.layout.njk" %}
+
+{% from "govuk/components/panel/macro.njk" import govukPanel %}
+
+{% set pageTitle = (appeal.siteVisitType | capitalize) + ' site visit booked' %}
+
+{% block pageHeading %}{% endblock %}
+{% block pageContent %}
+	{{
+    govukPanel({
```

Comment:
Indentation police 👮 - There is a mixture of space and tabs here.

I just can't help it.

Reply:
Shouldn't we set up a formatter for nunjucks? We have prettier for the js/sass but nothing for the html...